test/Microsoft.Sbom.Targets.Tests/AbstractGenerateSBomTaskInputTests.cs

```diff
@@ -15,18 +15,22 @@ namespace Microsoft.Sbom.Targets.Tests;
 [TestClass]
 public abstract class AbstractGenerateSBomTaskInputTests
```

Comment:
Can we rename this to AbstractGenerateSbomTaskInputTests (lowercase the B in SBom)? This is my bad, I accidentally wrote B instead of b